### PR TITLE
jawbone-updater: changed license to LGPL

### PR DESCRIPTION
**Pull request by gfrix**

### CODE REVIEW

File: Casks/jawbone-updater.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   url "http://content.jawbone.com/store/dashboard/Jawbone_Updater-#{version}.pkg"
   homepage 'http://jawbone.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :lgpl
```
Comment (vitorgalvao) [inline Casks/jawbone-updater.rb]: Not a valid value. `:gpl` should probably be used, as the closest approximation.

**Comment by Ftard5000** [comment]
The LGPL is a valid license, however. At some point in the future, LGPL should be recognized. For now, I'll just note it in a comment.

**Comment by vitorgalvao** [comment]
> I'll just note it in a comment.

Please don’t. The goal of `license` isn’t to be a legal note, it’s to provide a broad category. By your logic, every version of the GPL should have a value, but they don’t. [It’s done like that on purpose](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#license-stanza-details).

**Comment by Ftard5000** [comment]
Thank you for the information. I will not add a comment.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git) after?

**Comment by Ftard5000** [comment]
I"m sorry. I'm having problems squashing. Furthermore, edb4b20 doesn't appear to have any relation to the other commits, and I'm not sure how to retract it specifically.

**Comment by vitorgalvao** [comment]
No worries. Will be done in #8858. Your contribution is still included (and still credited to you).

**Pull request closed without merging**